templates/terraform/custom_expand/network_full_url.erb

```diff
@@ -0,0 +1,12 @@
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
```

Comment:
This looks really similar to self_link_from_name.erb. Does that expander work here?